rosidl_generator_dotnet/resource/msg.c.em

```diff
@@ -84,7 +84,10 @@ void @(msg_typename)__write_field_@(member.name)(void * message_handle, @(msg_ty
 @[        end if]@
```

Comment:
Suggestion: remove lines 63 through 76. This will collapse the logic to something like:

```
if( BasicType or String):
  <define read_field>
  <define write_field>
else:
  <define get_field>

Reply:
Ok. It seems to be working modified with your suggestion

Reply:
Cool. Otherwise the logic was:

```
if( BasicType or String):
  <define read_field>
else:
  // TODO: nested types not supported

if( BasicType or String):
  <define write_field>
else:
  <define get_field>
```

which feels a little weird. even if you got rid of the // TODO comment.

Reply:
Yes!!